wallet/src/model.rs

```diff
@@ -69,8 +69,7 @@ where
 #[derive(Debug, Serialize)]
 pub struct Transaction {
     pub hex_hash: String,
-    pub value: u64,
-    pub entry: TransactionEntry,
+    pub value: i64,
```

Comment:
Why is this now a `i64`? A `Transaction` value cannot be negative

Reply:
We got rid of the type `Entry` which was `Debit` or `Credit`, which now can be derived by Sheikah from its sign.

Reply:
Then this `value` is the change of the account balance? Add comment :)